index.js

```diff
@@ -9,17 +9,14 @@ const create = prod => (start, sink) => {
   let got2;
```

Comment:
remove this variable got2 since it won't be necessary

Reply:
oh, missed that, fixed